configure.ps1

```diff
@@ -145,6 +150,17 @@ $WithClouseau = $(If ($EnableClouseau) {1} else {0})
 $Hostname = [System.Net.Dns]::GetHostEntry([string]"localhost").HostName
 $WithProper = (-not $DisableProper).ToString().ToLower()
 $ErlangMD5 = ($EnableErlangMD5).ToString().ToLower()
+$WithSpiderMonkey = (-not $DisableSpiderMonkey).ToString().ToLower()
+
+if ($JSEngine -eq "quickjs") {
+    $WithSpiderMonkey = "false"
+}
+
+# If spidermonkey was disabled but JS_ENGINE set to "spidermonkey", reset it to "quickjs"
+if ( ($WithSpiderMonkey -eq "false" ) -and ($JS_ENGINE -eq "spidermonkey" ) ) {
+   Write-Verbose "NOTICE: Spidermonkey was disabled, but JS_ENGINE=spidermonkey. Setting JS_ENGINE=quickjs"
+   $JS_ENGINE = "quickjs"
+}
```

Comment:
Small nit / var names:

While Powershell isn't case-sensitive for variable names, let's use the same spelling.

```suggestion
if ( ($WithSpiderMonkey -eq "false" ) -and ($JSEngine -eq "spidermonkey" ) ) {
   Write-Verbose "NOTICE: Spidermonkey was disabled, but JSEngine=spidermonkey. Setting JSEngine=quickjs"
   $JSEngine = "quickjs"
}
```

Reply:
Good find, Ronny! This actually fixes an bug, as well, since in other parts of the config file we use `$JsEngine` already.